src/app/usecases/TextItemUseCase.js

```diff
@@ -0,0 +1,46 @@
+import {useState} from "react";
+import {getStringHash} from "../../utils/utils";
+
+
+const useTextItems = (initialTextItems = []) => {
+    const [textItems, setTextItems] = useState(initialTextItems);
+
+    const addTextItem = item => {
+        if (itemAlreadyExists(item, textItems) || isIllegalText(item)) {
+            console.log(`[addTextItem] item ${item} not added.`);
+            return;
+        }
+        /**
+         *  This is used to resolve the bug pointed out in issue#172.
+         *  See: https://stackoverflow.com/a/26254086/5290519.
+         */
```

Comment:
This can be removed.

Reply:
This is fine and clear, don't remove it.